daktari/checks/onepassword.py

```diff
@@ -41,12 +41,12 @@ def get_op_version() -> Optional[float]:
 
 class OPAccountExists(Check):
     depends_on = [OnePassInstalled]
+    name = "onepass.contextExists"
 
     def __init__(self, context_name: str):
         self.context_name = context_name
-        self.name = f"op.accountExists.{context_name}"
         self.suggestions = {
-            OS.GENERIC: f"<cmd>op signin {context_name}.onepassword.com <your-email-here></cmd>",
+            OS.GENERIC: f"<cmd>op signin {context_name}.1password.com <your-email-here></cmd>",
```

Comment:
Would context be something like `Glean`?

Reply:
Nope, we've not moved to Glean yet.

Reply:
oh sorry I just meant could that be an example? was trying to understand what context was 😄

Reply:
Sorry I totally misread with my first response :D